### PR TITLE
Task/3801/new_order_confirmation_view

### PR DESCRIPTION
**Pull request by tiwariav**
- separate views for 'order confirmation' and 'order details' under hosting app.
- fix for card brand name in 'order-confirmation' page

### CODE REVIEW

File: hosting/templates/hosting/order_confirm.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                <div class="modal-icon">
+                    <i class="fa fa-cog fa-spin fa-3x fa-fw"></i>
+                    <span class="sr-only">{% trans "Processing..." %}</span>
+                </div>
+                <h4 class="modal-title" id="createvm-modal-title"></h4>
+                <div class="modal-text" id="createvm-modal-body">
+                    {% trans "Hold tight, we are processing your request" %}
+                </div>
+                <div class="modal-footer">
+                    <a id="createvm-modal-done-btn" class="btn btn-success btn-ok btn-wide hide" href="{% url 'hosting:virtual_machines' %}">{% trans "OK" %}</a>
```
Comment (pcoder) [inline hosting/templates/hosting/order_confirm.html]: We can remove the href parameter here? I guess, in all cases, we redirect user according to server's response?

File: hosting/templates/hosting/order_confirm.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
+{% extends "hosting/base_short.html" %}
+{% load staticfiles i18n custom_tags %}
+
+{% block content %}
+<div id="order-detail{{order.pk}}" class="order-detail-container">
```
Comment (pcoder) [inline hosting/templates/hosting/order_confirm.html]: Not sure if `{{order.pk}}` serves any purpose in this case and same with setting the model in views.py as we have not created any HostingOrder yet

**Comment by pcoder** [comment]
Also, I find a different sentences in the payment page on hosting and datacenterlight. Not sure, if this is intentional?

> hosting: You are not making any payment yet. After submitting your card information, you will be taken to the Confirm Order Page
> datacenterlight: You are not making any payment yet. After placing your order, you will be taken to the Submit Payment Page

File: datacenterlight/templates/datacenterlight/order_confirm.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{% extends "hosting/order_confirm.html" %}
```
[the remark this answers is not in the document]
Comment (tiwariav) [reply datacenterlight/templates/datacenterlight/order_confirm.html]: yes, I've also thought over it, but currently the order and payment pages inherit styles directly from the `hosting/base_short.html` template. So I'll create a new ticket for this and work on it on a different PR?

**Comment by pcoder** [comment]
Wonder if we can update and merge this @tiwariav ?

**Comment by tiwariav** [comment]
Yes, the work intended was complete. But since its quite old now, I'll fix the conflicts and update the PR.

**Comment by tiwariav** [comment]
Closing this and continuing in PR #581

**Pull request closed without merging**